src/library/scala/collection/View.scala

```diff
@@ -165,48 +165,50 @@ object View extends IterableFactory[View] {
 
   /** A class that partitions an underlying collection into two views */
   @SerialVersionUID(3L)
+  @deprecated("Use Iterable.partition", "2.13.0")
   class Partition[A](val underlying: SomeIterableOps[A], val p: A => Boolean) extends Serializable {
```

Comment:
You can just remove it without deprecating it. The same goes for all the other classes.